src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,72 @@
+package calculator;
+
+import calculator.operator.Operator;
+import calculator.operator.OperatorFactory;
+
+import java.util.Arrays;
+
+public class Calculator {
+
+    private final static String FORMULA_SEPARATION_KEYWROD = " ";
+    private final static String[] OPERATOR_KEWORDS = {"+", "-", "*", "/"};
+
+    private String[] formula;
+
+    public Calculator(String input) {
+        validationEmptyWithInput(input);
+        this.formula = validationInvalidFormulaWithInput(input);
+    }
+
+    public Double calculate() {
+        double beforeNum = Double.parseDouble(formula[0]);
+
+        for (int i = 1; i < this.formula.length - 1; i=i+2) {
+            Operator operator = getOperator(formula[i]);
+            double afterNum = Double.parseDouble(formula[i+1]);
+            beforeNum = operator.calculate(beforeNum, afterNum);
+        }
+        return beforeNum;
+    }
+    
+    private Operator getOperator(String operatorString){
+        Operator operator = OperatorFactory.getOperator(operatorString);
+        if(operator == null){
+            throw new IllegalArgumentException("수식 중 사칙연산 기호가 잘못되었습니다.");
+        }
+        return operator;
```

Comment:
```suggestion
        return OperatorFactory.getOperator(operatorString);
```

예외처리는 분기흐름을 제어하는 측에 위임하는 것은 어떨까요.
사칙 연산 기호에 대한 정보가 집중되어 있는 쪽에서 담당하는 것이 자연스럽지 않을까요 

```
    public static Operator getOperator(String operatorString) {
        ...
        if (OPERATOR_KEYWORD_DIVISION.equals(operatorString)) {
            return new DivisionOperator();
        }

        throw new IllegalArgumentException("수식 중 사칙연산 기호가 잘못되었습니다.");
    }
```